File: examples/python/advanced/error-handling/test-inspect-error-details.py
@pytest.mark.durable_execution(
    handler=handler_with_error,
    lambda_function_name="error_function",
)
def test_error_details(durable_runner):
    """Test error details are captured."""
    with durable_runner:
        result = durable_runner.run(input={}, timeout=10)
    
    # Check error details
    assert result.status is InvocationStatus.FAILED
    assert result.error is not None
    assert "error_type" in result.error
    assert "message" in result.error

File: examples/java/advanced/error-handling/test-inspect-error-details.java
@Test
void testErrorDetails() {
    var handler = new ErrorHandler();
    var runner = LocalDurableTestRunner.create(Map.class, handler);

    var result = runner.runUntilComplete(Map.of());

    // Check error details
    assertEquals(ExecutionStatus.FAILED, result.getStatus());
    assertNotNull(result.getError());
    assertTrue(result.getError().getMessage().contains("RuntimeException"));
}
